examples/siro_sandbox/sandbox_app.py

```diff
@@ -121,6 +128,22 @@ def __init__(self, args, config, gui_input):
 
         self._cursor_style = None
 
+        self._sandbox_state = (
```

Comment:
As far as I remember, we agreed to play the tutorial at the beginning of each episode (see Eric's message [here](https://cvmlp.slack.com/archives/C0460NTKM4G/p1683922450164749?thread_ts=1683909138.185399&cid=C0460NTKM4G)). CC: @eundersander.

Thus, we need to have `self._show_tutorial = args.show_tutorial` attribute too. To know if we are in the "show tutorial" mode. Because, right now the app switches to `SandboxState.CONTROLLING_AGENT` at the end of the tutorial and tutorial doesn't start at the episode reset (pressing M key).